syncosaurus/syncosaurus.js

```diff
@@ -81,7 +86,6 @@ export default class Syncosaurus {
 
           //Reset keys in the case of `scan` method which can change keys that a subscriber pays attention to
           subscription.keys = newSubKeys;
```

Comment:
Should the possiblity of a scan method changing the keys being monitored mean that the query is fundamentally different, and should be treated differently? Like having another if/else branch from the if statement above, or having some type of flag or association between a scan method and its current keys?

Reply:
If I'm understanding your feedback correctly, I'm not sure that it's necessary to add another branch because keys are strings and replacing them even if the query doesn't contain a scan won't matter

Reply:
I removed this line of code because it is not necessary. A query with a `scan` is run every time based solely on whether or not the `scanFlag` property on the subscription object is true and not on the `keys` property.